newspaper/parsers.py

```diff
@@ -108,13 +109,18 @@ def getElementById(cls, node, idd):
 
     @classmethod
     def getElementsByTag(
-            cls, node, tag=None, attr=None, value=None, childs=False) -> list:
-        NS = "http://exslt.org/regular-expressions"
+            cls, node, tag=None, attr=None, value=None, childs=False, use_regex=False) -> list:
+        NS = None
         # selector = tag or '*'
         selector = 'descendant-or-self::%s' % (tag or '*')
         if attr and value:
-            selector = '%s[re:test(@%s, "%s", "i")]' % (selector, attr, value)
-        elems = node.xpath(selector, namespaces={"re": NS})
+            if use_regex:
+                NS = {"re": "http://exslt.org/regular-expressions"}
+                selector = '%s[re:test(@%s, "%s", "i")]' % (selector, attr, value)
+            else:
+                trans = 'translate(@%s, "%s", "%s")' % (attr, string.ascii_uppercase, string.ascii_lowercase)
```

Comment:
you are translating uppercase to lowercase before the contains check. Does that fully imitate the effects of the regex `re:test`?

Reply:
This immitates "i" flag in regular expression (case insensitive), which was used before.